src/main/java/com/launchcode/AroundTownServer/models/Event.java

```diff
@@ -0,0 +1,123 @@
+package com.launchcode.AroundTownServer.models;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import java.util.Objects;
+
+
+//import javax.validation.constraints.NotNull;
+
+@Entity
+public class Event {
+
+    private String name;
+
+    private String description;
+
+    private String location;
+
+    private String date;
+
+    private String time;
+
+    private String entryCost;
+
+    private boolean familyFriendly;
+
+    @Id
+    @GeneratedValue
+    public int eventId;
+
+
+    public Event(String name, String description, String location, String date, String time, String entryCost, boolean familyFriendly) {
+        this.name = name;
+        this.description = description;
+        this.location = location;
+        this.date = date;
+        this.time = time;
+        this.entryCost = entryCost;
+        this.familyFriendly = familyFriendly;
+    }
+
+    public Event() {
```

Comment:
By convention the empty constructor should be before the all arguments constructor.